### PR TITLE
Implements `getMessageFeeTokenID` - Closes #7566

### LINKED ISSUE #7566: Add getMessageFeeTokenID method to BaseInteroperabilityMethod
### Description

Add getMessageFeeTokenID method to BaseInteroperabilityMethod

```python
def getMessageFeeTokenID(chainID: ChainID) -> TokenID:
    # Check for direct channel
    if chainAccount(chainID) does not exist:
        chainID = CHAIN_ID_MAINCHAIN
    return channel(chainID).messageFeeTokenID
```

### Acceptance Criteria

- Should have all the relevant unit tests

### Additional Information

- Related LIP Update PR https://github.com/LiskHQ/lips/pull/148
- Depends on https://github.com/LiskHQ/lisk-sdk/issues/7565

### PR DESCRIPTION
**Pull request by Phanco**
### What was the problem?

This PR resolves #7566

### How was it solved?

A new function `getMessageFeeTokenID` is implemented

### How was it tested?

Updated unit-test

### CODE REVIEW

File: framework/src/modules/interoperability/base_interoperability_method.ts [hunk 1]
```diff
@@ -59,6 +60,16 @@ export abstract class BaseInteroperabilityMethod<
 		return this.getInteroperabilityStore(context).getTerminatedOutboxAccount(chainID);
 	}
 
+	public async getMessageFeeTokenID(
+		context: ImmutableMethodContext,
+		_chainID: Buffer,
+	): Promise<Buffer> {
+		const chainID = !(await this.getChainAccount(context, _chainID))
```
Comment (ishantiw) [inline framework/src/modules/interoperability/base_interoperability_method.ts:67]: maybe better to use,
```ts
this.stores.get(ChainAccountStore).has(chainID)
```

File: framework/src/modules/interoperability/base_interoperability_method.ts [hunk 2]
```diff
@@ -59,6 +60,16 @@ export abstract class BaseInteroperabilityMethod<
 		return this.getInteroperabilityStore(context).getTerminatedOutboxAccount(chainID);
 	}
 
+	public async getMessageFeeTokenID(
+		context: ImmutableMethodContext,
+		_chainID: Buffer,
```
Comment (ishantiw) [inline framework/src/modules/interoperability/base_interoperability_method.ts:65]: you can remove underscore `chain`

**Review by ishantiw — COMMENTED** [review]
Looks good, just a minor comment

**Review by ishantiw — APPROVED** [review]

**Comment by ishantiw** [comment]
yeah we can move the tests to base class and remove it from mainchain/sidechain as part of this issue https://github.com/LiskHQ/lisk-sdk/issues/7647

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**